docs/QUICKSTART.md

```diff
@@ -1,76 +1,190 @@
-- Install eraser from [deployment YAML](https://github.com/Azure/eraser/blob/v0.1.0/deploy/eraser.yaml).
-
-```bash
-$ kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v0.1.0/deploy/eraser.yaml
-namespace/eraser-system created
-customresourcedefinition.apiextensions.k8s.io/imagejobs.eraser.sh created
-customresourcedefinition.apiextensions.k8s.io/imagelists.eraser.sh created
-serviceaccount/eraser-controller-manager created
-role.rbac.authorization.k8s.io/eraser-leader-election-role created
-clusterrole.rbac.authorization.k8s.io/eraser-manager-role created
-rolebinding.rbac.authorization.k8s.io/eraser-leader-election-rolebinding created
-clusterrolebinding.rbac.authorization.k8s.io/eraser-manager-rolebinding created
-configmap/eraser-manager-config created
-deployment.apps/eraser-controller-manager created
-
-$ kubectl get pods -n eraser-system
-NAME                                         READY   STATUS    RESTARTS   AGE
-eraser-controller-manager-759c7c8794-q8qqg   1/1     Running   0          22s
-```
-
-- Create an [ImageList](./test/e2e/test-data/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove.
-
-Example:
-```bash
-cat <<EOF | kubectl apply -f -
-apiVersion: eraser.sh/v1alpha1
-kind: ImageList
-metadata:
-  name: imagelist
-spec:
-  images:
-    - sha256:2834dc507516af02784808c5f48b7cbe38b8ed5d0f4837f16e78d00deb7e7767
-    - docker.io/library/nginx:latest
-    - redis
-EOF
-```
+# Quickstart Tutorial
+
+1. Clone or create a new Codespace from the Eraser repo
+
+    ```shell
+    git clone https://github.com/Azure/eraser.git
+    cd eraser
+    ```
+    
+2. Create a cluster
```

Comment:
I think this should be optional -- there's the chance that someone will want to test with something other than Kind. 

Additionally, the e2e Kind cluster is accessible like the Eraser deployment yaml (https://raw.githubusercontent.com/Azure/eraser/v0.1.0/test/e2e/kind-config.yaml)

Reply:
I updated this to specify that for the purposes of this tutorial, a kind cluster will be used. When checking for the images on disk, we have to exec into the node, so if the user is running something other than docker for their nodes, then the command would not work. In addition, I added a note before the exec command so the user is aware of this. If we decide to completely omit the cluster creation, we should sync on how to demonstrate listing the images on disk.

I also inlined the kind-config file to not rely on the test data.